dawn/src/dawn/CodeGen/CXXNaive-ico/ASTStencilBody.cpp

```diff
@@ -73,11 +73,11 @@ void ASTStencilBody::visit(const std::shared_ptr<iir::ReductionOverNeighborExpr>
     }
   };
   std::string typeString = getLocationTypeString(expr->getRhsLocation());
-  ss_ << std::string(indent_, ' ') << "reduce" + typeString + "ToCell(LibTag{}, m_mesh, t, ";
+  ss_ << std::string(indent_, ' ') << "reduce" + typeString + "ToCell(LibTag{}, m_mesh, loc, ";
   expr->getInit()->accept(*this);
-  ss_ << ", [&](auto& lhs, auto const& t) { return lhs " << expr->getOp() << "= ";
+  ss_ << ", [&](auto& lhs, auto const& red_loc) { return lhs " << expr->getOp() << "= ";
```

Comment:
makes the generated code a bit more readable by having a different name inside the lambda (instead of shadowing the form `t` outside).